src/ORM/Search/BasicSearchContext.php

```diff
@@ -43,6 +43,7 @@ public function getQuery($searchParams, $sort = false, int|array|null $limit = n
         }
 
         $searchParams = $this->applySearchFilters($this->normaliseSearchParams($searchParams));
+        $this->setSearchParams($searchParams);
```

Comment:
Mirrors what happens in `SearchContext::getQuery()` as well.
I think I didn't add this originally because we don't use `$this->searchParams` directly in this class :sweat_smile: